inference-engine/tests/functional/plugin/conformance/test_runner/include/gflag_config.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <gflags/gflags.h>
+#include <iostream>
+
+static const char help_message[] = "Print a usage message.";
+static const char disable_test_config_message[] = "Required. Path to the Human Pose Estimation model (.xml) file.";
+static const char target_device_message[] = "Required. Specify the target device for Conformance Test Suite "
+                                            "(the list of available devices is shown below). Default value is CPU. "
+                                            "Use \"-d HETERO:<comma-separated_devices_list>\" format to specify HETERO plugin. "
+                                            "The application looks for a suitable plugin for the specified device.";
+static const char input_folders_message[] = "Required. Paths to the input folders with IRs";
+
+DEFINE_bool(h, false, help_message);
```

Comment:
I propose to follow unified arguments naming semantic, either full names or shorthands for every argument

Reply:
Done. But it's impossible to refine `--help`.